### ISSUE TITLE
Clean this up

### ISSUE THREAD

**Issue by tomaskontrimas** [icecube/skyllh#132]
https://github.com/icecube/skyllh/blob/dc58832f1dd2ead58675bb564575ce127aa7db65/skyllh/analyses/i3/publicdata_ps/utils.py#L388

**Comment by tomaskontrimas** [comment]
https://github.com/icecube/skyllh/blob/dc58832f1dd2ead58675bb564575ce127aa7db65/skyllh/analyses/i3/publicdata_ps/utils.py#L83

**Comment by tomaskontrimas** [comment]
https://github.com/icecube/skyllh/blob/dc58832f1dd2ead58675bb564575ce127aa7db65/skyllh/analyses/i3/publicdata_ps/utils.py#L518

**Comment by tomaskontrimas** [comment]
https://github.com/icecube/skyllh/blob/dc58832f1dd2ead58675bb564575ce127aa7db65/skyllh/analyses/i3/publicdata_ps/trad_ps_wMC.py

**Comment by chiarabellenghi** [comment]
Cleaned up everything but I did not remove the class for the second-order spline. It is used to create a spline of the effective area (which then is never used itself). I think it can be useful to keep a utility function to compute a 2-dimensional spline. I simply marked as "DEPRECATED" this method: https://github.com/icecube/skyllh/blob/409a3f26b8b97ee394a42c3d836312a560b22bb1/skyllh/analyses/i3/publicdata_ps/pd_aeff.py#L242
